manifests/v1beta1/components/controller/rbac.yaml

```diff
@@ -7,19 +7,38 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - configmaps
-      - serviceaccounts
       - services
       - events
-      - namespaces
+    verbs:
+      - "*"
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
       - persistentvolumes
       - persistentvolumeclaims
+    verbs:
+      - "get"
+      - "list"
+      - "watch"
+      - "create"
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
```

Comment:
Do we need permission for namespace in the Controller @johnugeorge @tenzen-y ?

Reply:
Yes. It is needed only in UI.

Reply:
I retested it. Currently, Katib needs get/list/watch for namespaces as well. Need to investigate separately